packages/fastify-vue/routing.js

```diff
@@ -113,7 +115,7 @@ export async function createRoute ({ client, errorHandler, route }, scope, confi
     const htmlPath = id.replace(/pages\/(.*?)\.vue$/, 'client/html/$1.html')
     const htmlSource = readFileSync(join(config.vite.root, config.vite.build.outDir, htmlPath), 'utf8')
     const htmlFunction = createHtmlFunction(htmlSource, scope, config)
-    handler = (_, reply) => htmlFunction.call(reply)
+    handler = (_, reply) => reply.html(htmlFunction)
```

Comment:
This is problematic, `reply.html()` needs to be `htmlFunction`. Let me see how can I fix this.